conf/config_ci_segmentation_local.yaml

```diff
@@ -52,12 +74,15 @@ training:
     rotate_limit: 45
     rotate_prob: 0.5
     hflip_prob: 0.5
+  normalization:
```

Comment:
Can you add some comments on the meaning of using it please

Reply:
sure